contact-tracing.postman_collection.json

```diff
@@ -784,6 +784,97 @@
 									"    },",
 									"    expiresAtTimestamp: {",
 									"      type: 'string'",
+									"    },",
+									"    smsSent: {",
+									"      type: 'boolean'",
+									"    }",
+									"  }",
+									"}",
+									"",
+									"const json = pm.response.json()",
+									"",
+									"pm.test('Status code is 200', () => {",
+									"  pm.response.to.have.status(200)",
+									"})",
+									"",
+									"pm.test('Schema is valid', () => {",
+									"  pm.expect(tv4.validate(json, schema)).to.be.true",
+									"})",
+									"",
+									"const control = CryptoJS.SHA512(json.code.toString().substr(0, Math.floor(json.code.length / 2))).toString(CryptoJS.enc.Hex)",
+									"const code = CryptoJS.SHA512(json.code.toString()).toString(CryptoJS.enc.Hex)",
+									"",
+									"pm.environment.set('uploadHash', `${control}${code}`)"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{pushToken}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"onsetDate\": \"2020-01-01\",\r\n    \"testDate\": \"2020-01-01\",\r\n    \"jobId\": \"{{$randomUUID}}\"\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								},
+								"urlencoded": {}
+							}
+						},
+						"url": {
+							"raw": "{{push}}/notify/positive",
+							"host": [
+								"{{push}}"
+							],
+							"path": [
+								"notify",
+								"positive"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Issue Verification Code",
```

Comment:
Looks like you've duplicated the `Notify positive` request with a different name - any reason we'd need both in the collection? Feel free to just rename the existing request to `Issue verification code`

Reply:
Actually I intended to update the endpoint to `/issue`. Looks like I forgot to export from postman after making that change.